pkg/trace/sampler/spansampler.go

```diff
@@ -0,0 +1,48 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
```

Comment:
nit: I'm not sure this warrants it's own file. It's probably fine to just put this single function into sampler.go. There are no global variables, new imports, or anything too long that would convolute rest of sampler.go s.t. this is needed in my opinion.